app/views/venues/edit.html.erb

```diff
@@ -1,10 +1,10 @@
-<h1>Editing a Venue</h1>
+<h3 style="text-align: center;" class="mt-5">Editing a Venue</h3>
```

Comment:
Don't use in line styling, use classes instead (bootstrap or your own).
`<h3 class="text-center">` uses bootstrap class to do the same